tools/filter_variant_SnpSift_cmd/filter_variant_SnpSift_cmd.xml

```diff
@@ -0,0 +1,109 @@
+<tool id="filer_variant_SnpSift_cmd" name="Generate filter line for variant filtering with SnpSift" version="0.1.0">
```

Comment:
<tool id="SnpSift_filter_builder" name="Build filter queries for SnpSift" version="0.1.0">

et renomme tout ce qui peut l'être par snpsift_filter_builder (nom du repo y compris, pas de -, _ à la place)

Reply:
ok je valide le nom pour ce repo